packages/kit/src/core/postbuild/analyse.js

```diff
@@ -121,14 +123,24 @@ async function analyse({
 		const prerender = page?.prerender ?? endpoint?.prerender;
 
 		if (prerender !== true) {
-			for (const feature of list_features(
+			for (const feature of list_route_features(
 				route,
 				manifest_data,
 				server_manifest,
 				tracked_features
 			)) {
 				check_feature(route.id, route_config, feature, config.adapter);
 			}
+
+			for (const additional of additional_entry_points) {
+				for (const feature of list_features(additional.file, server_manifest, tracked_features)) {
+					if (additional.disallowedFeatures?.includes(feature)) {
+						throw new Error(
+							`Usage of ${feature} (imported directly or indirectly) is not allowed in ${additional.file}`
```

Comment:
better to use a noun than an adjective for variable names like this

```suggestion
			for (const entry of additional_entry_points) {
				for (const feature of list_features(entry.file, server_manifest, tracked_features)) {
					if (entry.disallowedFeatures?.includes(feature)) {
						throw new Error(
							`Usage of ${feature} (imported directly or indirectly) is not allowed in ${entry.file}`
```